code/src/pages/PopularList.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+
+import MovieCard from "../components/MovieCard";
+
+const PopularList = () => {
+  const [popularList, setPopularList] = useState([]);
+  const API_URL =
```

Comment:
A suggestion is to store the  API's in their own file to reduce the amount of code in the components/pages files 🏷